tests/Integration/Behaviour/Features/Context/Domain/CartFeatureContext.php

```diff
@@ -42,6 +43,21 @@
 
 class CartFeatureContext extends AbstractDomainFeatureContext
 {
+    /**
+     * @given the current currency is :currencyIsoCode
+     */
+    public function addCurrencyToContext($currencyIsoCode)
+    {
+        $currency = new Currency();
+        $currency->name = $currencyIsoCode;
+        $currency->precision = 2;
```

Comment:
@matthieu-rolland The precision is always 2 ?

Reply:
For now yes, this method is only called in one place, if someone want to set another precision for the same feature test (which is unlikely for now), he can still change the method and make it go through a parameter.

Reply:
:ok_hand: No problem. I validate.